.github/workflows/on_push_branch__execute_ci_cd.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build_deps:
     runs-on: ubuntu-latest
-    container: bitwalker/alpine-elixir-phoenix:1.11
+    container: bitwalker/alpine-elixir-phoenix:1.13.1
```

Comment:
I am aware that there is no bitwalker/alpine-elixir-phoenix:1.14 . Looking deeper into this, it think we could refactor these github workflows to use our or other official elixir images. Of course, we will do this in a separate pull request.